### PR TITLE
Support of development/x major version

### PR DESCRIPTION
**Pull request by tcarmet**

### CODE REVIEW

File: bert_e/workflow/gitwaterflow/queueing.py [hunk 1]
```diff
@@ -104,14 +104,14 @@ def get_queue_branch(job, dev_branch: DevelopmentBranch, create=True
...
-    """Get the q/pr_id/x.y/* branch corresponding to a w/x.y/* branch."""
+    """Get the q/w/pr_id/x.y/* branch corresponding to a w/x.y/* branch."""
     wbranch_version = None
     if len(job.git.cascade.dst_branches) == 1 and \
        job.git.cascade.dst_branches[0].hfrev > 0:
         wbranch_version = job.git.cascade.dst_branches[0].version
     else:
         wbranch_version = wbranch.version
-    name = 'q/{}/{}/{}'.format(
+    name = 'q/w/{}/{}/{}'.format(
```
Comment (tcarmet) [inline bert_e/workflow/gitwaterflow/queueing.py:114]: breaking change here.

File: CHANGELOG [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
+## [4.0.0] - 2024-08-09
 # Removed
 - Support of tasks as it unused to due its incompatibility with GitHub.
 
 # Added
 - Bert-E's status notifications through a build status check.
+- Support of development branches with only major version: `development/x`.
+
+# Changed
+- Integration queue branches pattern is now `q/w/{pr.id}/{integration.branch}`
```
Comment (nootal) [inline CHANGELOG:13]: Let's make sure we communicate about that; I've looked around through https://github.com/search?q=org%3Ascality+q%2F+language%3Ayaml+path%3A%2F%5E%5C.github%5C%2F%2F&type=code and indeed I don't think it's going to be an issue.

File: bert_e/lib/versions.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+
```
Comment (nootal) [inline bert_e/lib/versions.py:2]:
```suggestion
```

**Review by nootal — CHANGES_REQUESTED** [review]
Only minor comment.
Quite impressive the amount of code for supporting this new feature.
Great job!

**Review by nootal — APPROVED** [review]

**Pull request merged**